Sources/KeyboardSupport/KeyboardToolbar.swift

```diff
@@ -8,7 +8,13 @@
 
 import UIKit
 
-/// Represents a toolbar shown above the keyboard.
+/// AutoNavigator will ask a TextInputView that conforms to this protocol for a toolbar to use in place of the default toolbar.
+/// If this variable is nil, the defualt toolbar will not be used.
```

Comment:
We may gain a little bit of clarity if we change the wording slightly (assuming this is correct).
```swift
/// If this variable is `nil`, then no toolbar will be used.
```

Reply:
Fixed. Should read ```"If this variable is nil, the default toolbar of the KeyboardAutoNavigator will be used."```

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/972bd57f1b7c1a26ff120e3ae5200a5cac499370